pulp_rpm/app/serializers/advisory.py

```diff
@@ -201,8 +201,13 @@ def validate(self, data):
         """
         update_record_data = dict()
         if "file" in data:
-            update_record_data.update(json.loads(data["file"].read()))
-            update_record_data.update(data)
+            try:
+                update_record_data.update(json.loads(data["file"].read()))
+                update_record_data.update(data)
```

Comment:
Does this line needs to be in the try/except clause?

Reply:
I don't think so.

Reply:
normally i would move this to the `else` clause however in this case this line does not pollute much, imo.